src/js/lib/DOMTranslation.ts

```diff
@@ -1125,6 +1131,10 @@ class DOMTranslation {
             this.addUserElementToCollection(translatableElements, element)
           }
         }
+        else if (mode === TranslationElementMode.ALL_ELEMENTS) {
+          forceVisibility = true;
```

Comment:
Ko šis dara?

Reply:
mode, kas atbild par to, ka tiek iztulkoti visi elementi, kas ir lapā, ne tikai tie, kas ir visible.
lai dziļi nemainīt loģiku, pa lielam visibility tiek forced, izmantojot jau eksistējošo property. jo šis tiešām arī darbojās un neprasīja baigās koda izmaiņas. pa lielam pie ALL_ELEMENTS mode visi lapas elementi tiek atzīmēti kā visible, un tiek padoti tulkošanai.

Reply:
Tas neizmaina izskatu lapā, ko redz gala lietotājs?

Reply:
Un kāpēc elementam jābūt redzamam, lai to tulkotu? Vai tad nav jau pārbaude, ka to neskatās, ja jaunais property ieslēgts?

Reply:
izskatu neietekmēja, scrollojot vienkārši teksti jau ir iztulkoti un nekāda tulkošana vairs nenotiek.
noņemot check uz visibility un noņemot forceVisibility, netulkojas <p> tagi. kāpēc? nav ne jausmas

Reply:
Paskatījos, tas tiešām ietekmē tikai to, vai tiek tulkots elements, nav sakara ar tā redzamības maiņu. Nosaukums laikam nāk no tā, ka iepriekš tulkoja tikai redzamos elementus. Varētu pārsaukt par `forceTranslation` vai kā tamlīdzīgi, vai arī komentāru pielikt par to, ko tas mainīgais dara.

Reply:
līdz šim izmantojās tikai vienuviet, pie visible elements tulkošanas, to force select un options taga tekstu iztulkošanu,
liekas ok pielietojums arī šeit, īpaši, ja viss labi un darbojās, darbosies